C++/include/algorithm/dynamic_programming/README.md

```diff
@@ -7,6 +7,12 @@ Dynamic programming algorithms are often used for optimization. A dynamic progra
 ### Contents
 
 1. [Matrix chain multiplication](#1-matrix-chain-multiplication)
+2. [Coin change](#2-coin-change)
+3. [0_1 Knapsack](#3-0_1-knapsack)
+4. [Kadane](#4-Kadane)
+5. [Longest Decreasing Subsequence](#5-Longest-Decreasing-Subsequence)
```

Comment:
"Longest decreasing subsequence" (only capitalize the first word in these titles – fix others, too)